<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>avahi.git - avahi</title>
<meta name='generator' content='cgit v0.10.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/avahi.git/atom/avahi-0.6.31-no-deprecations.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='avahi.git' href='/cgit/avahi.git/'>avahi.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f7'>f7</option>
<option value='f8'>f8</option>
<option value='f9'>f9</option>
<option value='master' selected='selected'>master</option>
<option value='olpc2'>olpc2</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>avahi</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/avahi.git/'>summary</a><a href='/cgit/avahi.git/refs/'>refs</a><a href='/cgit/avahi.git/log/avahi-0.6.31-no-deprecations.patch'>log</a><a class='active' href='/cgit/avahi.git/tree/avahi-0.6.31-no-deprecations.patch'>tree</a><a href='/cgit/avahi.git/commit/avahi-0.6.31-no-deprecations.patch'>commit</a><a href='/cgit/avahi.git/diff/avahi-0.6.31-no-deprecations.patch'>diff</a><a href='/cgit/avahi.git/stats/avahi-0.6.31-no-deprecations.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/avahi.git/log/avahi-0.6.31-no-deprecations.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/avahi.git/tree/'>root</a>/<a href='/cgit/avahi.git/tree/avahi-0.6.31-no-deprecations.patch'>avahi-0.6.31-no-deprecations.patch</a></div><div class='content'>blob: 9ca9d38dec7cfc91b5446d6f2da21c847703cb54 (<a href='/cgit/avahi.git/plain/avahi-0.6.31-no-deprecations.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
</pre></td>
<td class='lines'><pre><code>--- avahi-0.6.31/avahi-ui/Makefile.am.orig	2013-09-21 16:47:28.677659361 +0200
+++ avahi-0.6.31/avahi-ui/Makefile.am	2013-09-21 16:47:34.852520537 +0200
@@ -15,7 +15,7 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307
 # USA.
 
-AM_CFLAGS=-I$(top_srcdir) -DG_DISABLE_DEPRECATED=1 -DGDK_DISABLE_DEPRECATED=1 -DGTK_DISABLE_DEPRECATED=1
+AM_CFLAGS=-I$(top_srcdir)
 
 # This cool debug trap works on i386/gcc only
 AM_CFLAGS+='-DDEBUG_TRAP=__asm__("int $$3")'
--- avahi-0.6.31/configure.ac.orig	2013-09-21 16:46:43.750669413 +0200
+++ avahi-0.6.31/configure.ac	2013-09-21 16:47:02.566246400 +0200
@@ -92,7 +92,7 @@
     AC_MSG_CHECKING([whether stack-smashing protection is available])
     ssp_old_cflags="$CFLAGS"
     ssp_old_ldflags="$LDFLAGS"
-    CFLAGS="$CFLAGS -Werror -fstack-protector-all -fPIC"
+    CFLAGS="$CFLAGS -fstack-protector-all -fPIC"
     LDFLAGS="$LDFLAGS -Wl,-z,defs"
     cat confdefs.h &gt; conftest.c
     cat &gt;&gt;conftest.c &lt;&lt;_ACEOF
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.1 at 2014-05-14 13:49:24 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
